runtime/server/x86/bin/decoder_main.cc

```diff
@@ -1,17 +1,16 @@
 // Copyright 2020 Mobvoi Inc. All Rights Reserved.
 // Author: binbinzhang@mobvoi.com (Binbin Zhang)
 
-#include <glog/logging.h>
-#include <gflags/gflags.h>
-
-#include <torch/torch.h>
-#include <torch/script.h>
+#include "glog/logging.h"
```

Comment:
Please sort them by tools, not.by.hands.

Reply:
okay, fixed by clang-format